pages/index.tsx

```diff
@@ -1,282 +1,30 @@
 import React from 'react'
-import { useMediaQuery } from 'react-responsive'
-import styled from 'styled-components'
 import Link from 'next/link'
-import { GetStaticProps } from 'next'
-import { selectors } from '@defencedigital/design-tokens'
-import { IconFileDownload } from '@defencedigital/icon-library'
 
-import { Footer } from '../components/presenters/Docs/Footer'
-import { FooterExternalLink } from '../components/presenters/Docs/Footer/FooterExternalLink'
-import { PageBanner } from '../components/presenters/Docs/PageBanner'
-import {
-  Masthead,
-  MastheadMenu,
-  MastheadMenuItem,
-  MastheadSubMenu,
-  MastheadSubMenuItem,
-} from '../components/presenters/Docs/Masthead'
-import { Badge, BADGE_VARIANT } from '../components/presenters/Docs/Badge'
-import { LayoutHomepage } from '../components/layouts/Homepage'
-import { contentful } from '../services/contentful'
-import HOMEPAGE_BY_ID_QUERY from '../graphql/queries/HomepageByID.graphql'
-import NAVIGATION_BY_ID_QUERY from '../graphql/queries/NavigationByID.graphql'
-import {
-  Navigation as NavigationType,
-  Homepage as HomepageType,
-} from '../graphql'
+import {Footer} from '../components/presenters/Docs/Footer'
```

Comment:
There should be white space here either side of the brackets - is Prettier doing something weird?

Reply:
Yes. This must've been Webstorm's default formatting